ignite3/src/main/java/site/ycsb/db/ignite3/IgniteJdbcClient.java

```diff
@@ -26,6 +26,35 @@ public class IgniteJdbcClient extends AbstractSqlClient {
    */
   private static final ThreadLocal<Connection> CONN = new ThreadLocal<>();
 
+  /** Prepared statement for reading values. */
+  private static final ThreadLocal<PreparedStatement> READ_PREPARED_STATEMENT = ThreadLocal
+      .withInitial(IgniteJdbcClient::buildReadStatement);
+
+  /** Prepared statement for inserting values. */
+  private static final ThreadLocal<PreparedStatement> INSERT_PREPARED_STATEMENT = ThreadLocal
+      .withInitial(IgniteJdbcClient::buildInsertStatement);
+
+  /** Build prepared statement for reading values. */
+  private static PreparedStatement buildReadStatement() {
+    try {
+      return CONN.get().prepareStatement(readPreparedStatementString);
+    } catch (SQLException e) {
+      LOG.error("Unable to prepare statement for SQL: " + readPreparedStatementString, e);
+      return null;
```

Comment:
I guess we should throw some kind of `RuntimeException` here. Instead we would simply get NPE when trying to use result from this method.

Reply:
updated